src/k8s-engine/wsl.ts

```diff
@@ -390,15 +390,33 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     return this.#desiredPort;
   }
 
+  /**
+   * List the registered WSL2 distributions.
+   */
   protected async registeredDistros({ runningOnly = false } = {}): Promise<string[]> {
-    const args = ['--list', '--quiet'];
+    const stdout = await this.execWSL({ capture: true }, '--list', '--verbose');
+    // As wsl.exe may be localized, don't check state here.
+    const parser = /^[\s\*]+(?<name>.*?)\s+\w+\s+(?<version>\d+)\s*$/;
```

Comment:
IDE linter complaint that the backslash before `*` is redundant.

Reply:
Fixed in 6e784ca8ee4c3e109b72a7f2c870f76299fc6e54.